### ISSUE TITLE
intron numbering - missing warning

### ISSUE THREAD

**Issue by jtdendunnen** [mutalyzer/mutalyzer#525]
Variant description NC_000023.11(NM_004006.2):c.1149+412A>G is not correct (it should be c.1150-239G>A). Mutalyzer's Position Converter gives NC_000023.11:g.32644552C>T and corrects the description in the listing below. The Name Checker gives "0 Errors, 0 Warnings". A warning should be given by the Name Checker and probably best also by the Position Converter.

**Comment by jfjlaros** [comment]
Indeed, a warning should be raised here.

[Mutalyzer 3](https://v3.mutalyzer.nl/namechecker/NC_000023.11(NM_004006.3):c.1149+412A%3EG) does mention that a correction was performed. By clicking on the details (the triangle at the right of the blue bar), the reason for the correction can be seen.

Is this satisfactory?

**Comment by jtdendunnen** [comment]
Yes, although I think the warning could be clearer. Something like: nucleotide position corrected from '1149+412' to '1150-239; nucleotide numbering changes from + to - in the centre of the intron.

**Comment by jfjlaros** [comment]
Agreed, but Mutalyzer 3 does not correct a description in this sense. It applies an interpretable description to the reference sequence to obtain a new sequence and extracts the corrected description by comparing the reference sequence with this new sequence. It does not correct the individual parts of a description separately.

What we can do, is to link to a help page with commonly occurring corrections, explanations and links to the relevant sections of the HGVS nomenclature website.
